packages/fx-core/src/plugins/resource/bot/progressBars.ts

```diff
@@ -32,6 +33,18 @@ export class ProgressBarFactory {
     return handler;
   }
 
+  public static async createBarFromContext(
```

Comment:
Why not use existing `newProgressBar`? I think that method is also compatible to `v2.Context`.

Reply:
OK, also changed the old behavior to avoid returning a progress bar whose status is ended.